### ISSUE TITLE
Is it possible to set a stating numberOfLines / textView height?

### ISSUE THREAD

**Issue by kevinrenskers** [slackhq/SlackTextViewController#121]
When I know that a certain instance of SlackTextViewController will be used to write longer pieces of text (news posts rather then chat messages) it would be nice to have the textview start off with a bigger version. That looks more inviting then the one single line. 

How can I make this work, or would it require changes in SlackTextViewController?

**Comment by dzenbot** [comment]
The `numberOfLines` is a dynamic value calculated with `self.contentSize.height/self.font.lineHeight`.
Although, it you have a look at `slk_minimumInputbarHeight`, you could achieve what you're looking for by returning a higher number.
This is a private method tho, so you'll need to fork and extend, and probably fix a few layout issues that will come along (ie: the counter doesn't show up when there is only 1 line of text; the text starts from the top to the bottom, this might look weird so adjusting the `textContainerInset` could look better; etc.)

PS: Sorry for the late response!

![image](https://cloud.githubusercontent.com/assets/590579/6343277/ad32fbb8-bbc4-11e4-8f5c-2d3be7178e71.png)

**Comment by kevinrenskers** [comment]
Alright, can I make a feature request so this would be more natively supported? :)

**Comment by dzenbot** [comment]
This feature is more of a specific requirement, which makes it not generic enough to be supported natively. I would recommend you to try what I suggested, but beware of rare bugs since things might get a bit messy because of this tweak.

**Comment by JGVB** [comment]
I've set the `textContainerInset` to have a larger `top` inset.   

Initially, it looks ok:
<img width="372" alt="screen shot 2017-05-10 at 9 41 45 am" src="https://cloud.githubusercontent.com/assets/3453556/25901498/f926950e-3564-11e7-9659-3afe281c6df4.png">

But when text is typed, the left and right buttons rise up, and the textView calculates `numberOfLines` as 3. Which is because the textView's `contentSize.height` is 34 (which may be 1 or 2 lines), then it executes;
`contentHeight -= self.textContainerInset.top + self.textContainerInset.bottom;`
  `NSUInteger lines = fabs(contentHeight/self.font.lineHeight);`
The large `self.textContainerInset.top` is added to the content height, and number of lines is calculated to be 3..which is 2 too many :(

<img width="373" alt="screen shot 2017-05-10 at 9 48 59 am" src="https://cloud.githubusercontent.com/assets/3453556/25901828/ec91fee0-3565-11e7-8991-1046dab0c912.png">


Also, I have issues when I set the `textContainerInset` top inset to smaller, then back to larger again.  I've put in `layoutSubviews()` in all spots that it seems to make sense.
